ratpack-core/src/main/java/ratpack/http/client/internal/DefaultHttpClient.java

```diff
@@ -194,12 +227,14 @@ public HttpClientSpec readTimeout(Duration readTimeout) {
 
   @Override
   public Promise<ReceivedResponse> request(URI uri, final Action<? super RequestSpec> requestConfigurer) {
-    return Promise.async(downstream -> new ContentAggregatingRequestAction(uri, this, 0, Execution.current(), requestConfigurer).connect(downstream));
+    Promise<ReceivedResponse> p =  Promise.async(downstream -> new ContentAggregatingRequestAction(uri, this, 0, Execution.current(), requestConfigurer.append(spec.requestInterceptor)).connect(downstream));
+    return p.next(spec.responseInterceptor);
```

Comment:
IDEA wouldn't let me string these 2 lines together. It wouldn't figure out the typing.